### ISSUE TITLE
Files containing multi byte characters get served with wrong Content-Length on "grunt server"

### ISSUE THREAD

**Issue by nnarhinen** [yeoman/generator-webapp_DEPRECATED#13]
A file like

``` html
<h1>Foo</h1>
<p>Multibyte characters here: ääää ööööö åååååå</p>

<p>Some html here</p>
```

The output gets truncated. I tried to isolate the problem with standalone "send"-module usage, standalone "connect"-module usage and standalone "grunt-contrib-connect" usage but didn't succeed so I guess it has to be something with the generated Gruntfile or something.

The most curious thing is, the Content-Length has wrong value only on GET requests. On HEAD it's the right one (bytesize by fs.stat).

**Comment by passy** [comment]
Which Node version and OS do you use?

**Comment by nnarhinen** [comment]
Ok so I found the root cause when digging some more.. the reason was grunt-contrib-livereload and pull request is already made:

gruntjs/grunt-contrib-livereload#10

**Comment by sindresorhus** [comment]
Thanks. Included in grunt-contrib-livereload 0.1.2
